atom/browser/api/atom_api_url_request.cc

```diff
@@ -125,6 +125,20 @@ bool URLRequest::ResponseState::Failed() const {
   return IsFlagSet(ResponseStateFlags::kFailed);
 }
 
+mate::Dictionary URLRequest::GetUploadProgress(v8::Isolate* isolate) {
+  mate::Dictionary progress = mate::Dictionary::CreateEmpty(isolate);
+
+  if (!atom_request_) {
+    progress.Set("active", false);
+    return progress;
+  } else {
+    progress.Set("active", true);
+  }
+
+  atom_request_->GetUploadProgress(progress);
+  return progress;
```

Comment:
_(minor)_ This control flow is a little awkward; e.g. the use of an `else` block after a `return`. Lines 135 and 138 go together.

More readable flow:

```cpp
mate::Dictionary progress = ...
if (atom_request_) {
  progress.Set("active", true);
  atom_request_->GetUploadProgress(progress);
} else {
  progress.Set("active", false);
}
return progress;
```